subiquity/cloudinit.py

```diff
@@ -1,12 +1,20 @@
 """Shared cloudinit utility functions"""
 
 import json
+import logging
+
+log = logging.getLogger("subiquity.cloudinit")
 
 
 def get_host_combined_cloud_config() -> dict:
     """Return the host system /run/cloud-init/combined-cloud-config.json"""
     try:
         with open("/run/cloud-init/combined-cloud-config.json") as fp:
-            return json.load(fp)
+            config = json.load(fp)
+            log.debug(
+                "Loaded cloud config from /run/cloud-init/combined-cloud-config.json"
+            )
+            return config
     except (IOError, OSError, AttributeError, json.decoder.JSONDecodeError):
+        log.debug("Failed to load combined-cloud-config")
```

Comment:
```suggestion
        log.debug("Failed to load combined-cloud-config")
```
```suggestion
    except (IOError, OSError, AttributeError, json.decoder.JSONDecodeError) as e:
        log.debug("Failed to load combined-cloud-config: %s", e)
```

Reply:
I think that's fine at debug level, wouldn't want anything too scary looking as this is expected behavior for existing install ISOs to not have the file.
I'll follow up.

Reply:
good point!

Reply:
https://github.com/canonical/subiquity/pull/1743